.storybook/recipes/GlobalHeader/GlobalHeader.module.css

```diff
@@ -42,27 +42,41 @@
  *    on small screens, but is hidden when the nav
  *    can be exposed
  */
-.global-header__menu-button {
-  @media all and (min-width: $eds-bp-lg) {
+.global-header__menu-button.global-header__menu-button {
+  transition: background var(--eds-anim-fade-quick) var(--eds-anim-ease);
+
+  &:hover,
+  &:focus {
+    background: var(--eds-color-brand-grape-800);
+  }
+
+  @media all and (min-width: $eds-bp-xl) {
     display: none; /* 1 */
   }
 }
 
-
 /**
- * Avatar block 
- * 1) The bottom-aligned avatar block contains the clickable
- *    avatar item that exposes the user's notifications when
- *    selected
- * 2) Nudge the alignment to achieve the desired layout
+ * Global header utility nav
  */
-.global-header__avatar-block {
+.global-header__utility-nav {
   position: relative;
-  left: -0.5rem; /* 2 */
   margin-left: auto;
 
-  @media all and (min-width: $eds-bp-lg) {
+  @media all and (min-width: $eds-bp-xl) {
+    left: -0.5rem;
```

Comment:
the logo & utility nav look a little too far left of the PrimaryNav:
<img width="160" alt="Screen Shot 2022-04-07 at 10 59 08 AM" src="https://user-images.githubusercontent.com/15840841/162230140-72b66559-ac5b-4c16-a7f7-1d84fc9f3288.png">

Is this intentional? Or should we remove the `left:` styling from them?

Reply:
I added some padding for the hover states to the primary nav items. I can tighten this alignment stuff up